website/blog/2018-08-27-7.0.0.md

```diff
@@ -0,0 +1,444 @@
+module.exports = {
+  plugins: [
+    env === "production" && "babel-plugin-that-is-cool"
+  ].filter(Boolean)
+};
+```
+
+`babel.config.js` has a different config resolution than a `.babelrc`. It will always resolve the config from that file versus originally when Babel would do a lookup from each file upward until it found a config. This makes it possible to take advantage of the next feature posted below, `overrides`.
+
+## Selective Configuration with `overrides`
+
+Recently, I published a [post](https://babeljs.io/blog/2018/06/26/on-consuming-and-publishing-es2015+-packages) with thoughts on both publishing ES2015+ packages and also consuming/compiling them.
+
+There's a [section](https://babeljs.io/blog/2018/06/26/on-consuming-and-publishing-es2015+-packages#selective-compilation-with-overrides) that goes into using a new key in Babel's config called `overrides` which allows you to specify different configs per glob.
+
+```javascript=
+module.exports = {
+  presets: [
+    // defeault config...
+  ],
+  overrides: [{
+    test: ["./node_modules"],
+    presets: [
+      // config for node_modules
+    ],
+  }, {
+    test: ["./tests"],
+    presets: [
+      // config for tests
+    ],
+  }]
+};
+```
+
+This allows an application that requires different compilation configs for its tests, client code, and server code to skip needing to create a new `.babelrc` file per folder.
+
+## Speed 🏎
+
+Babel itself is [faster](https://twitter.com/left_pad/status/927554660508028929) so it should take less time to build! We've made a lot of changes to optimize the code as well as [accept](https://twitter.com/bmeurer/status/924346958810107904) [patches](https://twitter.com/rauchg/status/924349334346276864) from the [v8](https://twitter.com/v8js) team. We're glad to be a part of the [Web Tooling Benchmark](https://github.com/v8/web-tooling-benchmark) alongside many other great JavaScript tools.
+
+### Output Options
+
+Babel has supported [preset and plugin options](https://babeljs.io/docs/en/next/plugins#plugin-options) for some time now. To recap you can wrap the plugin in an array and pass an options object to the plugin:
+
+```diff
+{
+  "plugins": [
+-   "pluginA",
++   ["pluginA", {
++     // options here
++   }],
+  ]
+}
+```
+
+We've made some changes to the `loose` option of some plugins and added some new options for others! Note by using these options you are opting into non-spec compliant behavior and should know what you are doing; this can become an issue when you switch off of compiling to use the syntax natively. These kinds of options are best used in a library, if at all.
+
+- For classes, `class A {}` will now not include the `classCallCheck` helper.
+
+```javascript
+class A {}
+```
+
+```diff
+var A = function A() {
+-  _classCallCheck(this, A);
+};
+```
+
+- There is a new option if every use of a `for-of` loop is just an array: `["transform-for-of", { "assumeArray": true }]`
+
+```javascript
+let elm;
+for (elm of array) {
```

Comment:
this shouldn't matter, but at first i thought you were talking about the language elm, not element :) Maybe just `let el;`

Reply:
haha I think I copied it from a test